### ISSUE TITLE
[Win]最新版 (2.8.4) 的最小化/恢复窗口在某些多标签程序中存在bug

### ISSUE THREAD

**Issue by JohnKhanJr** [yingDev/WGestures2-mac-bugs#120]
比如在XnView Classic或Photoshop中，执行最小化/恢复窗口手势其实是最小化了程序内的标签窗口，需要再执行一次手势才最会真正的最小化程序主窗口

**Comment by yingDev** [comment]
这倒不是bug, 毕竟子窗口也是窗口. 有人或许有这种需求.
不过, 上个版本的  窗口控制 差点加上了这个选项:

- [ ] 允许作用于子窗口

也许下个版本会加上.

**Comment by superDMS** [comment]
对,我也发现了,更新之后, 反而没之前好用了,好多程序出现子窗口最小化,,,,,  ....还是原来的逻辑好,,,全局最小化适用面更广  反而是作用于子窗口是少数情况  ,毕竟很多程序做子窗口是察觉不出来的, 用户默认是全局窗口 ,,,,子窗口更应该作为特殊选项...我还以为是我的错觉....原来是真的改了逻辑,,,,大多程序的设计逻辑都是一窗一程序,即使是有子窗口 ,但用户都会把它当成一体的,,,,建议加子选项,或者回滚原来的逻辑,,,.
